### DISCUSSION TITLE
Improve Tasks conversion into Issue

### DISCUSSION THREAD

**Discussion by alx-so** [community/community#6898] [Projects and Issues]
## Steps to reproduce

1. Create issue - attach it to some project and milestone
2. Set some task list
3. Submit issue
4. Convert task to issue
![Screenshot (1)](https://user-images.githubusercontent.com/8975621/139243064-7fc5440c-25b1-4c53-91d4-df8758574388.png)
5. Result: sub-issue is created, and attached to parent issue. But it is not attached to the same Project/Milestone as parent issue.

## Suggestion

IMHO, would be great and very convenient, if by default:

1. by converting task to issue, Project/Milestone is also copied to newly created task
2. by closing parent issue, all task issues, that tracked, is also closing. Because you have such situation, when parent issue is closed, but tasks not:
![Capture3](https://user-images.githubusercontent.com/8975621/139243958-37c9d601-564d-4562-ad8b-401cc5da053e.PNG)

**Comment by nexushoratio** [comment]
I would like to see this apply to all metadata of the parent issue, in my case, particularly labels and assignees.

Personally, I would be fine with a pop-up confirmation that had the fields to be propagated checked by default (or, eventually, per project setting).

- #4330
- #12172
